azure-functions-maven-plugin/src/test/java/com/microsoft/azure/maven/function/DeployMojoTest.java

```diff
@@ -240,16 +240,16 @@ public void getMSDeployArtifactHandler() throws MojoExecutionException {
         assertTrue(handler instanceof MSDeployArtifactHandlerImpl);
     }
 
-    @Test
-    public void getFTPArtifactHandler() throws MojoExecutionException {
-        doReturn("azure-functions-maven-plugin").when(mojoSpy).getPluginName();
-        doReturn("test-path").when(mojoSpy).getBuildDirectoryAbsolutePath();
-        doReturn(DeploymentType.FTP).when(mojoSpy).getDeploymentType();
-        final ArtifactHandler handler = mojoSpy.getArtifactHandler();
-
-        assertNotNull(handler);
-        assertTrue(handler instanceof FTPArtifactHandlerImpl);
-    }
+//    @Test
+//    public void getFTPArtifactHandler() throws MojoExecutionException {
```

Comment:
why remove this case?

Reply:
Commit by mistake, fixed in later commit.